runPairingKey.go

```diff
@@ -16,7 +16,11 @@ func pairingKeyNew(argv map[string]interface{}) (err error) {
 	index := LoadIndex(fsAPI, org)
 
 	id := NewID()
-	key := hex.EncodeToString(crypto.RandomBytes(16))
+	random, err := crypto.RandomBytes(16)
+	if err != nil {
+		panic(fmt.Sprintf("Couldn't get random bytes: %s", err.Error()))
```

Comment:
The error  returned by RandomBytes() function already says basically the exact same thing. I think we may want to reconsider our conventions on where this kind of context is placed. 

Also, err.String() calls err.Error() and fmt.Sprintf() implicitly calls err.String, so there isn't a need to call err.Error(). Could just do:
panic(fmt.Sprintf("Couldn't get random bytes: %s", err))